simulator/modules/writeback/writeback.cpp

```diff
@@ -43,7 +43,7 @@ auto Writeback<ISA>::read_instructions( Cycle cycle)
 {
     auto ports = { rp_branch_datapath.get(), rp_mem_datapath.get(), rp_execute_datapath.get() };
     std::vector<Instr> result;
-    result.reserve( 1); // depends on WB througput
+    result.reserve( bandwidth);
```

Comment:
Actually, we do not need `bandwidth` field here any longer. `std::vector` reservation may be set to `ports.size()`.

Reply:
Got it.